src/commands/deploy/deploy.ts

```diff
@@ -88,6 +95,26 @@ async function deploy(actionContext: IActionContext, arg1: vscode.Uri | string |
     await notifyDeployComplete(context, node, context.workspaceFolder.uri.fsPath);
 }
 
+async function updateWorkerProcessTo64BitIfRequired(context: IDeployContext, siteConfig: WebSiteManagementModels.SiteConfigResource, node: SlotTreeItemBase, language: ProjectLanguage): Promise<void> {
+    const functionProject: string | undefined = await tryGetFunctionProjectRoot(context.workspaceFolder.uri.fsPath);
+    if (functionProject === undefined) {
+        throw new Error(localize('failedToFindFuncHost', 'Unable not locate Azure Functions project.'));
+    }
+    const projectFiles: string[] = await dotnetUtils.getProjFiles(language, functionProject);
+    if (projectFiles.length === 0) {
+        throw new Error(localize('unableToFindProj', 'Unable to detect project file.'));
+    }
+    const mainProject: string = path.join(functionProject, projectFiles[0]);
+    const platformTarget: string | undefined = await dotnetUtils.getPlatformTarget(mainProject);
+    if (platformTarget === 'x64' && siteConfig.use32BitWorkerProcess === true) {
+        const config: WebSiteManagementModels.SiteConfigResource = {
+            use32BitWorkerProcess: false
```

Comment:
Does this preserve all existing properties on the site config? Or does it change them back to the default? I think it would be safer if you edited the existing `siteConfig` object and sent that in `updateConfiguration`

Reply:
My intent here is to only update app to 64 bit and affect nothing else. If I  use existing `siteConfig` and somebody mutates `siteConfig` object, then I will unintentionally update other settings as well.

I can use existing `siteConfig` object if you believe it is safe to do so.